pallets/invulnerables/src/migration.rs

```diff
@@ -0,0 +1,91 @@
+// Copyright (C) Moondance Labs Ltd.
+// This file is part of Tanssi.
+
+// Tanssi is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Tanssi is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Tanssi.  If not, see <http://www.gnu.org/licenses/>
+
+//! A module that is responsible for migration of storage for Invulnerables.
+
+use super::*;
+use frame_support::{log, traits::OnRuntimeUpgrade};
+
+/// Version 1 Migration
+/// This migration ensures that any existing `Invulnerables` storage lists are sorted.
+pub mod v1 {
+    use super::*;
+    use frame_support::pallet_prelude::*;
+    #[cfg(feature = "try-runtime")]
+    use sp_std::prelude::*;
+
+    pub struct MigrateToV1<T>(sp_std::marker::PhantomData<T>);
+    impl<T: Config> OnRuntimeUpgrade for MigrateToV1<T> {
```

Comment:
I dont understand this migraiton. At the beginning there wont be any invulnerables. What we need to do is set them for the first time, but that has nothing to do with the storage version of the pallet (as it did not change). You can remove this fiole for now, as we need to do the migration in some other way

Reply:
I assumed we would work on it together. You can ignore it for now.